packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.types.ts

```diff
@@ -40,7 +40,7 @@ export interface IChoiceGroupProps extends React.InputHTMLAttributes<HTMLElement
 
   /**
    * Deprecated and will be removed by 07/17/2017 Use 'onChange' instead.
-   * @deprecated
+   * @deprecated Use 'onChange' instead.
```

Comment:
>'onChange' ins [](start = 21, length = 14)

Use backticks or `{@link IChoiceGroupProps.onChange}`

Reply:
Our current documentation parser doesn't support `{@link IChoiceGroupProps.onChange}`. We have some inconsistencies in our documentation as to where backticks are used - I can updated the @deprecated messages to use backticks for now and we can revisit the overall backtick issue.